index.html

```diff
@@ -8,29 +8,149 @@
   <link rel="stylesheet" href="main.css">
 </head>
 <body>
-  <header>
+  <header id="top">
     <a href="#"><img src="./pics/LogoPlaceholder.png" alt="logo"></a>
     <nav>
       <a href="#"><img src="./pics/Icon-Menu.png" alt="menu"></a>
     </nav>
   </header>
   <main>
-    <section>
+    <section id="headline">
       <h1>Hey there. I'm Edozie. <br> <span id="title-span">I'm a software developer</span></h1>
-      <p id="text">
+      <p class="text">
         I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
       </p>
       <ul class="socials">
-        <li><a href="#"><img src="/pics/github.png" alt="github logo"></a></li>
-        <li><a href="#"><img src="/pics/linkedin.png" alt="linkedin logo"></a></li>
-        <li><a href="#"><img src="/pics/Enabled.png" alt="enabled"></a></li>
-        <li><a href="#"><img src="/pics/twitter.png" alt="twitter"></a></li>
-        <li><a href="#"><img src="/pics/medium.png" alt="medium"></a></li>
+        <li><a href="#"><img src="./pics/github.png" alt="github logo"></a></li>
+        <li><a href="#"><img src="./pics/linkedin.png" alt="linkedin logo"></a></li>
+        <li><a href="#"><img src="./pics/Enabled.png" alt="enabled"></a></li>
+        <li><a href="#"><img src="./pics/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="./pics/medium.png" alt="medium"></a></li>
       </ul>
       <div >
-        <a href="#" ><img src="/pics/Action.png" alt="action button"></a>
+        <a href="#"><img src="./pics/Action.png" alt="action button"></a>
       </div>
     </section>
+    <section id="works">
+      <header id="recent">
+        <h1>My Recent Works</h1>
```

Comment:
Title of a page? On line 34? This is one page, these are just different sections of the same page. I'm confused now.

Reply:
Please @topeogunleye, are you saying I should change the h1 in line 36 to h2? I don't understand the compulsory suggestion you made, so I don't worsen my situation due to time. Thanks.

Reply:
Yes, that's what I'm saying.

Reply:
Okay, thanks boss.

Reply:
One of the articles Microverse sent us said you can use h1 within every article tag. I have tried to change it, but it's messing up other stuff in my code. Since it's optional, I will focus on the compulsory one, and then change it later. Thanks @topeogunleye 🙏🏾

Reply:
@julzedz. I hope It's clear enough